packages/react-components/react-migration-v0-v9/src/components/Button/index.tsx

```diff
@@ -0,0 +1 @@
+export { buttonMigrationStyles } from './ButtonMigration.mixins';
```

Comment:
For better tree shaking it would be better to do:

```ts
// ButtonMigration.mixins.ts
export const v0Icon = /* ... */
export const v9DisabledCursor = /* ... */
```

And then:

```ts
// index.tsx
export * as buttonMigrationStyles from './ButtonMigration.mixins';
```

Or are there problems with API extractor with this approach?

Reply:
Done